deploy-0chain/action.yml

```diff
@@ -378,23 +383,23 @@ runs:
       run: |
         timeout 300 ./stake_blobbers.sh
         if [[ $? -ne 0 ]]
-          then
-            cat build.log
-            cd ..
-            echo '#!/bin/bash
-            RED="\033[0;31m"
-            NC="\033[0m"
-            echo
-            echo -e "${RED}======================================================${NC}"
-            echo -e "${RED}RETRYING DEPLOYMENT ($(echo $1 | awk '"'"'{ print toupper($0) }'"'"') ATTEMPT):${NC}"
-            echo -e "${RED}======================================================${NC}"
-            echo
-            ./teardown.sh && ./deploy.sh && cd $GITHUB_WORKSPACE && ./stake_blobbers.sh $1;
-            ' > ./retry.sh;
-            chmod 777 ./retry.sh;
-            timeout 300 ./retry.sh "second" || cd $GITHUB_WORKSPACE && cat build.log && cd .. && timeout 300 ./retry.sh "final" || cd $GITHUB_WORKSPACE && cat build.log && exit 1;
-          else
-            echo "Blobbers staked succefully on try $((ITER+1)). Moving on to tests..."
+        then
+          cat build.log
+          cd ..
+          echo '#!/bin/bash
+          RED="\033[0;31m"
+          NC="\033[0m"
+          echo
+          echo -e "${RED}======================================================${NC}"
+          echo -e "${RED}RETRYING DEPLOYMENT ($(echo $1 | awk '"'"'{ print toupper($0) }'"'"') ATTEMPT):${NC}"
+          echo -e "${RED}======================================================${NC}"
+          echo
+          ./teardown.sh && ./deploy.sh && cd $GITHUB_WORKSPACE && ./stake_blobbers.sh $1;
+          ' > ./retry.sh;
+          chmod 777 ./retry.sh;
+          timeout 300 ./retry.sh "second" || cd $GITHUB_WORKSPACE && cat build.log && cd .. && timeout 300 ./retry.sh "final" || cd $GITHUB_WORKSPACE && cat build.log && exit 1;
+        else
+          echo "Blobbers staked succefully on try $((ITER+1)). Moving on to tests..."
         fi
 
     - name: "Extract Logging From Pods"
```

Comment:
I added to the if below in my PR
https://github.com/0chain/actions/pull/56/files#diff-f4b0c0a5824ad0fb33cd15ecc1fbc29fa819ad9d948948b7c8f9eec5fbc00459R384
so we only do this if we have deployed a network out as part of the test run (as opposed to for example running against dev)

Reply:
fixed